setup.py

```diff
@@ -33,6 +33,7 @@
         ],
     },
     install_requires=[
+        'wheel',
```

Comment:
This doesn't make sense here, pretty much all Python packages depend on `wheel`, if this is missing there is something wrong with your setup.

Reply:
I'm not the only one to complain about this issue.

Reply:
But I don't really know what to do to debug it.